README.md

```diff
@@ -3,8 +3,27 @@
 This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
 
 ## Setup
-- `git clone https://github.com/thewca/worldcubeassociation.org` - Clone this repo! (And navigate into it, `cd worldcubeassociation.org`)
-- `(cd WcaOnRails; bundle install && bundle exec pre-commit install) && git config pre-commit.ruby "scripts/ruby_in_wca_on_rails.sh"` - Set up git pre-commit hook. Optional, but very useful.
+- Clone this repo! (And navigate into it)
+  ```
+  git clone https://github.com/thewca/worldcubeassociation.org
+  cd worldcubeassociation.org
+  ```
+- Ensure you have the correct [Ruby version](./.ruby_version) installed. We recommend using a Ruby version manager like [rvm](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv). They should both read `.ruby_version` file to use the correct version (`rvm current` or `rbenv version` to confirm).
```

Comment:
Not exactly sure what happened with my previous suggestion, but the slight typo is still there (the file is `.ruby-version` instead of `.ruby_version`).

```suggestion
- Ensure you have the correct [Ruby version](./.ruby-version) installed. We recommend using a Ruby version manager like [rvm](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv). They should both read the `.ruby-version` file to use the correct version (`rvm current` or `rbenv version` to confirm).
```

Reply:
Strange. I thought I committed it. Must have overridden it myself. Should be fixed now. Thanks!